app/src/main/java/net/aiscope/gdd_app/ui/main/MainActivity.kt

```diff
@@ -36,6 +38,9 @@ class MainActivity : AppCompatActivity(), SelectDiseaseView, LogoutFLow {
         setContentView(R.layout.activity_main)
         setSupportActionBar(toolbar)
 
+        @SuppressLint("SetTextI18n")
+        main_version_text.text = "v${BuildConfig.VERSION_NAME}"
```

Comment:
What do you think about using a formatted string resource instead of hardcoding the string literal here?
See https://developer.android.com/guide/topics/resources/string-resource.html#formatting-strings

Reply:
I didn't know about that possibility - much better! Thanks!!